ptx/S_12_flux_int.ptx

```diff
@@ -551,7 +551,8 @@
         </p>
       </task>
   </activity>
-
+ <p>
+  Look to Hughes Hallet for a note on Orientation of surfaces.
```

Comment:
There's a missing `</p>` here that's causing this section not to compile properly. Is this a note of something to do during editing or something that you've got WIP on but haven't put in, @nelong?

Reply:
This is a something to do later note. We talked about needed to put some mention of orientability in this section (likely as an aside but maybe a section at the end). I think HughesHallet has a similar section to use as inspiration.

Reply:
I put in the </p>.